From 43e66ffa7f33e03b9d15faff025fe3b4f536280e Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Sat, 28 Mar 2020 17:26:29 +1100
Subject: [PATCH] winmm: Use _lopen instead of OpenFile.

OpenFile has a limit of a filepath limit of 128.
Use _lopen which calls through to CreateFile which
has a greater limit.
---
 dlls/winmm/mmio.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/dlls/winmm/mmio.c b/dlls/winmm/mmio.c
index 6f143019751..30adcd0f228 100644
--- a/dlls/winmm/mmio.c
+++ b/dlls/winmm/mmio.c
@@ -76,8 +76,7 @@ static LRESULT CALLBACK mmioDosIOProc(LPMMIOINFO lpmmioinfo, UINT uMessage,
 
 	    /* if filename NULL, assume open file handle in adwInfo[0] */
 	    if (szFileName) {
-                OFSTRUCT    ofs;
-                lpmmioinfo->adwInfo[0] = OpenFile(szFileName, &ofs, lpmmioinfo->dwFlags & 0xFFFF);
+                lpmmioinfo->adwInfo[0] = _lopen(szFileName, lpmmioinfo->dwFlags);
             }
 	    if (lpmmioinfo->adwInfo[0] == HFILE_ERROR)
 		ret = MMIOERR_FILENOTFOUND;
-- 
2.25.1

